tests/template_tests/test_parser.py

```diff
@@ -73,14 +72,6 @@ def test_variable_parsing(self):
         with six.assertRaisesRegex(self, TypeError, "Variable must be a string or number, got <(class|type) 'dict'>"):
             Variable({})
 
-    @override_settings(DEBUG=True)
-    def test_compile_filter_error(self):
-        # regression test for #19819
-        msg = "Could not parse the remainder: '@bar' from 'foo@bar'"
-        with six.assertRaisesRegex(self, TemplateSyntaxError, msg) as cm:
-            Template("{% if 1 %}{{ foo@bar }}{% endif %}")
-        self.assertEqual(cm.exception.django_template_source[1], (10, 23))
-
```

Comment:
I moved this test to ``template_tests.tests.TemplateTests``. I think it's more appropriate there since it's also testing ``Template`` and not just ``Parser``.